### ISSUE TITLE
Make /info/bidders endpoint consistent between Go and Java

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#1788]
As discussed in PBS committee, there's currently a difference between PBS-Go and PBS-Java in the /info/bidders endpoint. 

- PBS-Go lists all bid adapters whether or not they're enabled
- PBS-Java lists only enabled adapters

As both use cases are useful, the proposal is to add a query string parameter (?enabledonly=true/false). This value has no default and will return HTTP 400 if not provided with body "missing enabledonly flag"

The results of the /info/bidders/BIDDER endpoint should include all the fields currently in the PBS-Go output:

```
{
    "status": "ACTIVE",
    "usesHttps": true,
    "maintainer": {
        "email": "scope.sspp@adform.com"
    },
    "capabilities": {
        "app": {
            "mediaTypes": [
                "banner",
                "video"
            ]
        },
        "site": {
            "mediaTypes": [
                "banner",
                "video"
            ]
        }
    }
}
```

**Comment by SyntaxNode** [comment]
Sounds good to me. What should the behavior be if enabledonly is not to either true or false?  Perhaps we should use the default, or another option is to return an error.

**Comment by SyntaxNode** [comment]
Discussed with a few other folks. We currently prefer to validate `enabledonly` and return an error if it's not true or false, case insensitive.

**Comment by bretg** [comment]
this was released with PBS-Java 1.71

**Comment by SyntaxNode** [comment]
I implemented this for PBS-Go, but noticed the specifications had since changed to return a 400 if the new `enabledonly` is not provided. This is a breaking change which will affect folks at Xandr.  To avoid problems for existing usages, I will have `enabledonly` default to `false` for PBS-Go.

**Comment by bretg** [comment]
This is ok @SyntaxNode , but the spec changing was due to your request from April 7th. :-)

**Comment by SyntaxNode** [comment]
Ah! I see. In that case I believe this is a misunderstanding. I didn't intend to make it required, but to validate the parameter only when its provided, such that:

/info/bidders -> ok (backwards compability)
/info/bidders?enabledonly=false -> all bidders (current default in PBS-Go)
/info/bidders?enabledonly=true -> enabled only
/info/bidders?enabledonly= -> 400 error
/info/bidders?enabledonly=foo -> 400 error

**Comment by SyntaxNode** [comment]
Implemented in PBS-Go 0.177.0.
